ballerina/Dependencies.toml

```diff
@@ -22,7 +22,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "data.jsondata"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
Do we need this version changes

Reply:
Since 0.2.0 already released, we need to version up with new fix, don't we? Since this is new feature do we need to as this into 1.0.0 or 0.3.0?

Reply:
We can bump it to 0.3.0 with this new feature

Ideally, we need this feature in a stable version like 1.0.0 (with Update 11) in order to use it in http package. @SasinduDilshara we are releasing a stable version with update 11 right?

Reply:
Yeah, we can do it